cmd/oceantv/broadcast_states.go

```diff
@@ -633,7 +633,7 @@ func onFailureClosure(ctx *broadcastContext, cfg *BroadcastConfig) func(err erro
 				_cfg.StartFailures++
 				// TODO: make this configurable in config.
 				const maxStartFailures = 3
-				if _cfg.StartFailures >= maxStartFailures {
+				if disableOnFirstFail || _cfg.StartFailures >= maxStartFailures {
 					_cfg.StartFailures = 0
 					_cfg.Enabled = false
 					notifier.Send(
```

Comment:
I'm thinking maybe we could supply the disableOnFirstFail status to the notify message + the last error that occured (which is a cheeky inclusion into this PR).

Reply:
I have included the disableOnFirstFailure status to the message now. I don't think there is a good way to also send the last error to occur without changing all calls to onfailure closure, so I think that should be a different PR.

Reply:
The error is provided as a parameter to the returned closure already, so you can use that.